lib/engine/game/g_1835/meta.rb

```diff
@@ -16,6 +16,13 @@ module Meta
         GAME_RULES_URL = 'http://google.com'
 
         PLAYER_RANGE = [3, 7].freeze
+        OPTIONAL_RULES = [
+          {
+            sym: :Clemens,
+            short_name: 'Clemens',
+            desc: 'all Privates and minors are available, 4-3-2-1-1-2-3-4, Minors start when Bay 50%',
```

Comment:
presumably "Buy" instead of "Bay" ?

Reply:
no, minors will only start, if the Bayern was floated.

Reply:
Ah is that the Bayrische Eisenbahn?

Reply:
yes.

Reply:
May want to use the full name, for clarity